src/torchmetrics/detection/ciou.py

```diff
@@ -0,0 +1,61 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Callable, Optional
+
+from torch import Tensor
+
+from torchmetrics.detection.iou import IntersectionOverUnion
+from torchmetrics.functional.detection.ciou import _ciou_compute, _ciou_update
+from torchmetrics.utilities.imports import _TORCHVISION_GREATER_EQUAL_0_8, _TORCHVISION_GREATER_EQUAL_0_13
+
+if _TORCHVISION_GREATER_EQUAL_0_8:
+    from torchvision.ops import box_convert
+else:
+    box_convert = None
+    __doctest_skip__ = ["CompleteIntersectionOverUnion"]
```

Comment:
there is no doctest, so this is not needed

Reply:
ups